index.js

```diff
@@ -51,7 +59,152 @@ module.exports = function (app) {
     return new Date(new Date().getTime() - 60 * 60 * 1000).toISOString()
   }
 
-  return {
+  function getQuery(startTime, endTime, path) {
```

Comment:
Strings are so like arrays that maybe the parameter should be named `pathElements` or `pathArray`? TypeScript, anyone....